calculation/seistech_calc/seistech_calc/plots/plt_plotting.py

```diff
@@ -1225,3 +1237,21 @@ def _get_causal_params_bounds(cs_param_bounds: gms.CausalParamBounds):
             "vs30": cs_param_bounds.site_info.db_vs30,
         },
     }
+
+
+def _calc_cdf(weights: List, x_values: List):
```

Comment:
Don't think its worth making a function for this if its only used once.
Also you can sort a dataframe by its columns directly, i.e. in line 1005 becomes
`contribution_df = contribution_df.sort_values()` and then lines 1019 and 1020 become
`contribution_df.index.values` & `contribution_df.values`
Also I suggest renaming `contribution_df` to `contribution_series` since its actually a pd.Series not pd.DataFrame 
Also passing numpy arrays to matplotlib is perfectly fine, so no need to cast to list as you do in line 1019 & 1020

Reply:
Apparently, it is actually `pd.DataFrame`

As suggested, used the following code which is still a DatFrame.
```python
cs_param_bounds.contr_df.loc[
    :, ["contribution", "magnitude"]
].set_index("magnitude", drop=True)
```                

![image](https://user-images.githubusercontent.com/7849113/133709506-2251f5b2-89d1-4a81-942c-7ce7121a8a3c.png)